OpenTrons/opentrons-api
# fix(components): fix storybook deck configurator

# Overview
fix storybook deck configurator

close RAUT-862

![Screenshot 2023-12-12 at 4 59 34 PM](https://github.com/Opentrons/opentrons/assets/474225/21d72913-a29f-476b-81ed-e114ceef9a24)

![Screenshot 2023-12-12 at 4 59 42 PM](https://github.com/Opentrons/opentrons/assets/474225/13ab7a6a-595a-4c9b-853c-c7595f421c26)



# Test Plan
1. run storybook
2. check followings
- http://localhost:6060/?path=/story/library-molecules-simulation-deckconfigurator--default
- http://localhost:6060/?path=/story/library-molecules-simulation-deckconfigurator--read-only


# Changelog



# Review requests



# Risk assessment
low
